### PR TITLE
gst-plugins-rs 0.9.8

### PR DESCRIPTION
**Pull request by redoPop**
Created with `brew bump-formula-pr`.

### CODE REVIEW

File: Formula/gst-plugins-rs.rb [hunk 1]
```diff
@@ -22,14 +21,19 @@ class GstPluginsRs < Formula
...
+  depends_on "python@3.11" # for tomli
 
-  # commit ref, https://gitlab.freedesktop.org/gstreamer/gst-plugins-rs/-/commit/ea98a0b5964cd196abbb48c621969a8ef33eb157
-  # remove in next release
-  patch :DATA
+  # used by meson dependency checker to parse cargo files
+  resource "tomli" do
+    url "https://files.pythonhosted.org/packages/c0/3f/d7af728f075fb08564c5949a9c95e44352e23dee646869fa104a3b2060a3/tomli-2.0.1.tar.gz"
+    sha256 "de526c12914f0c550d15924c62d72abc48d6fe7364aa87328337a31007fe8a4f"
+  end
```
Comment (bayandin) [inline Formula/gst-plugins-rs.rb:36]: I don't think there's a usage of this resource in the formula

Comment (redoPop) [reply Formula/gst-plugins-rs.rb]: Yep, looks like it uses tomllib by default and only falls back to tomli for older Pythons. Fixed in 777be6d892f0c752aecdf6814127b13314e33d7e. ty!

**Comment by redoPop** [comment]
From the Ubuntu failure [in the latest CI build:](https://github.com/Homebrew/homebrew-core/actions/runs/4084930329/jobs/7042329456)

> ../meson.build:77:2: ERROR: Dependency 'gstreamer-gl-1.0' is required but not found.

Not sure what to make of this. gstreamer-gl is part of gst-plugins-base, which is already a dependency. I'll see if I can set up an ubuntu box later to replicate this (my dev machine is an M1 Pro / Ventura, which builds just fine) but if anyone has a hunch in the meantime I'm happy to try it.

**Comment by redoPop** [comment]
@bayandin I don't have permission to add labels to this PR, but need some help here:

I've set up an old Intel Mac Mini with Ubuntu 22.04.1 LTS, checked out the PR locally, and confirmed that `brew edit gst-plugins-rs` reveals the updated formula.

My problem: when I run `brew install --debug --build-from-source gst-plugins-rs` Homebrew downloads and attempts to build the current master (0.8.4) formula instead of the updated local formula. Is an additional flag needed in Linux Homebrew?

**Comment by bayandin** [comment]
> My problem: when I run brew install --debug --build-from-source gst-plugins-rs Homebrew downloads and attempts to build the current master (0.8.4) formula instead of the updated local formula. Is an additional flag needed in Linux Homebrew?

Try to enable developer mode (`brew developer on`) and run your command (`brew install -s ...`) again.

**Comment by bayandin** [comment]
Also, you could try `HOMEBREW_NO_INSTALL_FROM_API=1 brew install -s ...`

**Comment by redoPop** [comment]
Thanks, exactly what I needed!

I can now reproduce this error and also confirm that `gst-plugins-base` on linuxbrew is missing the `gstreamer-gl-1.0` dependency. `--verbose reinstall` corroborates this.

Digging further: `gstreamer-gl-1.0` is OpenGL-specific. The `gst-plugins-base` package has no OpenGL equivalent in its Linux dependencies, so its builds skip `gstreamer-gl-1.0` in Linux. Adding `freeglut` to its package deps resolves that issue, after which `gst-plugins-rs` is able to to complete builds.

I'll have to circle back later tonight, but my plan is to open a new PR that adds `freeglut` to the linux dependencies block of `gst-plugins-base` and thus make the `gstreamer-gl-1.0` library available for this PR.

**Comment by redoPop** [comment]
Looks like the last CI build failure was a temporary network issue.

**Review by bayandin — COMMENTED** [review]
Nice work!

Does `python@3.11` required in runtime, or it's only a build dependency? If so, it should be replaced with `depends_on "python@3.11" => :build`

Also, could you please rebase and squash all your commits into one with the message `gst-plugins-rs 0.9.8`

**Review by bayandin — APPROVED** [review]
Looks great, thanks @redoPop 🎉

**Pull request closed without merging**